docker-compose/docker-compose.yaml

```diff
@@ -90,6 +141,7 @@ services:
       - 'JAEGER_SERVER_URL=http://jaeger:16686'
       - 'GITHUB_BASE_URL=http://github-proxy:3180'
       - 'PROMETHEUS_URL=http://prometheus:9090'
+      - 'NEW_MIGRATIONS=true'
```

Comment:
These NEW_MIGRATIONS flags should be removed, right? We'll use them for testing until https://github.com/sourcegraph/sourcegraph/pull/29144 gets merged the but they shouldn't be included in this PR.

Reply:
Correct. Removed!

Reply:
Reminder that this change snuck back in during CI testing and should be removed